src/Shared/FileUtilities.cs

```diff
@@ -393,6 +398,48 @@ private static bool IsUNCPath(string path)
             return isUNC || path.IndexOf(@"\\?\globalroot", StringComparison.OrdinalIgnoreCase) != -1;
         }
 
+#if !CLR2COMPATIBILITY
+        /// <summary>
+        /// Gets the exact case used on the file system for an existing file or directory.
+        /// </summary>
+        /// <param name="path">A relative or absolute path.</param>
+        /// <returns>The full path using the correct case if the path exists. Otherwise, the full path.</returns>
+        private static string GetExactPath(string path)
```

Comment:
This doesn't do the same thing as GetFullPath right? AFAIK GetFullPath also removes relative dir parts. And I don't remember whether it does anything about slash normalization like collapsing multiple slashes.